ulib/FStar.BV.fst

```diff
@@ -90,6 +90,11 @@ let bvdiv #n a b =
 let int2bv_div #n #x #y #z pf =
   inverse_vec_lemma #n (bvdiv #n (int2bv #n x) y)
 
+let bvdiv_unsafe #n a b = admit ()
```

Comment:
I know I said you could admit it, but on second thoughts, since this just file serves a model for bv_t, you could give a model for this operation by testing if `b` is zero and if so returning anything (b, for example), and otherwise calling bvdiv a b.

Reply:
provided a model with `x/0 = 0`.